src/Styles/Nav.scss

```diff
@@ -0,0 +1,37 @@
+@import 'Mixins';
+
+ul,
+li {
+   list-style-type: none;
+   margin: 0;
+   padding: 0;
+}
+
+.navBar {
+  // height: 50px;
+  background: rgb(156, 171, 196);
+  padding: 0 5px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #072E73;
+}
+
+.navList{
+  display: flex;
+  width: 10%;
+  justify-content: space-around;
+  font-size: 30px;
+}
+
+
```

Comment:
I'm curious as to how the `px` are working (or not working) with different screen sizes?

Reply:
@SallyHaefling got the impression from previous project feedback from you all that it was important to have consistency across the project.  I'd been using a combination of rems and px, after a suggestion from Travis last mod that sometimes px are more appropriate for things like small padding and margins because they're more predictable.  But we've since changed everything to pixels (with the exception of percentages for widths) for consistency's sake.  Would you be so kind as to clarify best practice on this @thatPamIAm?